makefile

```diff
@@ -0,0 +1,36 @@
+# Compiler
+CXX = g++
+
+# Compiler flags
+CXXFLAGS = -lprotobuf
+
+# Source files
+EXAMPLE_DIR = Example
```

Comment:
gali šitą makefile tiesiog perkelti į example folderį, tuomet nereikės šito dalyko